README.md

```diff
@@ -22,6 +24,20 @@ cd terraform/
 tfenv install
 ```
 
+To use the CodeQL resuable workflow, add the following job to the jobs section of you CI workflow:
+
+```
+sast:
```

Comment:
Could the choice of name be problematic for if/when acquire Snyk in future? At that point we'll presumably have two sast tasks - we should choose a name that's forwards-compatible.

Reply:
Yes I agree, I think a more descriptive name would be appropriate. I'll change it now.